cfgov/unprocessed/apps/youth-employment-success/js/route-option-view.js

```diff
@@ -0,0 +1,106 @@
+import { checkDom, setInitFlag } from '../../../js/modules/util/atomic-helpers';
+import inputView from './input-view';
+
+const CLASSES = Object.freeze( {
+  FORM: 'o-yes-route-option',
+  TRANSPORTATION_CHECKBOX: 'a-yes-route-mode',
+  QUESTION_INPUT: 'a-yes-question'
+} );
+
+const defaultState = {
+  selectedTransportation: [],
+  daysPerWeek: '',
+  miles: '',
+  dailyCost: ''
+};
+
+/**
+ * RouteOptionFormView
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element
+ *  The root DOM element for this view
+ * @returns {Object} The view's public methods
+ */
+function RouteOptionFormView( element ) {
+  let state = { ...defaultState };
+
+  const _dom = checkDom( element, CLASSES.FORM );
+  const _transportationTypes = Array.prototype.slice.call(
+    _dom.querySelectorAll( `.${ CLASSES.TRANSPORTATION_CHECKBOX }` )
+  );
+  const _textInputs = Array.prototype.slice.call(
+    _dom.querySelectorAll( `.${ CLASSES.QUESTION_INPUT }` )
+  );
+
+  /**
+   * Updates form state from child <input text> nodes
```

Comment:
Should this say `<input> text nodes`?

Reply:
Maybe? I was just short-handing `<input type="text">` and am willing to change it to whatever you think is more clear 😄